apps/astarte_pairing/lib/astarte_pairing_web/controllers/device_controller.ex

```diff
@@ -47,6 +49,18 @@ defmodule Astarte.PairingWeb.DeviceController do
       |> put_status(:created)
       |> put_view(CredentialsView)
       |> render("show_astarte_mqtt_v1.json", credentials: credentials)
+    else
+      {:error, reason} ->
+        Logger.info("Failure to create credentials.",
+          event: "CredentialsFailure",
+          realm: realm,
+          hw_id: hw_id,
+          module: __MODULE__,
+          function: "create_credentials/2",
+          reason: inspect(reason)
```

Comment:
these are a lot of metadata

some are not even accepted by logger (check loggers configuration in the config files), but none are required. let's limit these to the bare minimum


```suggestion
          realm: realm,
          hw_id: hw_id
```